src/System.ServiceModel.Duplex/pkg/System.ServiceModel.Duplex.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.ServiceModel.Duplex.csproj">
-      <SupportedFramework>net45;win8;netcoreapp1.0;$(AllXamarinFrameworks)</SupportedFramework>
+      <SupportedFramework>net45;win8;netcoreapp2.0;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
Ok, so leaving netcoreapp1.0 means we support the higher versions as well, in this case netcoreapp1.1 and netcoreapp2.0, yes?
In your email you mentioned that we 

> should update the SupportedFrameworks you expect cover with your build

 so how do we need to update them then?